scc/cert/aggregated_signature.go

```diff
@@ -14,6 +14,20 @@ type AggregatedSignature[S Statement] struct {
 	signature bls.Signature
 }
 
+func NewAggregatedSignature[S Statement](
+	signers BitSet[scc.MemberId],
+	signature bls.Signature) AggregatedSignature[S] {
+	return AggregatedSignature[S]{signers: signers, signature: signature}
```

Comment:
Should document that the arguments are not deep-copied.

Reply:
The signature and bitset is indeed deep copied. I will add documentation and test to reflect this.